packages/form-validations/README.md

```diff
@@ -126,4 +192,49 @@ _Get the errors of the form or a specific field_
 | Parameter | Default | Required? | Description                                                                                                                                                                                                                                                            |
 | --------- | ------- | --------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | input     | -       | ✕         | When `input` is given, it returns the errors of that input element, otherwise returns all errors of the form as an object, using input element as key and corresponding errors as value. `validate()` must be called before expecting this method to return correctly. |
-```
+
+**FormValidations.setLocale(locale)**
```

Comment:
I think it would be nicer to have this as a title? Or add one more empty line before.